inject-generator/src/main/java/io/avaje/inject/generator/Processor.java

```diff
@@ -143,6 +123,21 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     return false;
   }
 
+  private static Set<TypeElement> importedElements(RoundEnvironment roundEnv) {
+    return roundEnv.getElementsAnnotatedWith(element(ImportPrism.PRISM_TYPE)).stream()
+        .map(ImportPrism::getInstanceOn)
+        .flatMap(p -> p.value().stream())
+        .map(ProcessingContext::asElement)
+        .peek(e -> addImportedType(e.getQualifiedName().toString()))
```

Comment:
Last I checked `Stream.peek` has a tendency to skip over elements in certain situations

Reply:
Hmm, I've used peek() a decent amount and not hit a skipping issue? You want it changed back?

Reply:
better safe than sorry